components/admin/analytics/revenue-metrics.component.tsx

```
@@ -0,0 +1,159 @@
+ * @param value - Numeric value to format
+ * @returns Formatted percentage string
+ */
+function formatPercentage(value: number): string {
+  return `${value.toFixed(1)}%`;
+}
+
+/**
+ * Revenue metrics component.
+ * Displays key revenue and subscription metrics in card format.
+ */
+export function RevenueMetrics({ data }: RevenueMetricsProps) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'RevenueMetrics'. Consider to delete it or rename it.

'RevenueMetrics' is defined here:


(lint/suspicious/noRedeclare)